### ISSUE TITLE
OpenAI compatible API support

### ISSUE THREAD

**Issue by matbee-eth** [AmberSahdev/Open-Interface#18]
```
Traceback (most recent call last):
  File "app/app.py", line 74, in <module>
    app = App()
          ^^^^^
  File "app/app.py", line 39, in __init__
    self.core = Core()
                ^^^^^^
  File "/tmp/_MEIS364J5/core.py", line 22, in __init__
    self.llm = LLM()
               ^^^^^
  File "/tmp/_MEIS364J5/llm.py", line 59, in __init__
    self.model = ModelFactory.create_model(self.model_name, base_url, api_key, context)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/_MEIS364J5/models/factory.py", line 13, in create_model
    raise ValueError(f'Unsupported model type {model_name}. Create entry in app/models/')
ValueError: Unsupported model type Qwen/Qwen2-VL-7B-Instruct-AWQ. Create entry in app/models/
[1431460] Failed to execute script 'app' due to unhandled exception!

```

**Comment by matbee-eth** [comment]
Can't open the app at all, just errors and doesnt launch

**Comment by AmberSahdev** [comment]
Oh! I don't remember why I wrote it that way - let me fix over the weekend

**Comment by ludvigpujsek** [comment]
Same issue but with _llava_ model running through _ollama_

**Comment by ludvigpujsek** [comment]
After digging through the temporary file generated and following the python functions back to where the settings are read:
The file that causes the program to no longer launch properly is a settings.json saved in _%userprofile%/.open-interface_ or (most probably) _c:/Users/<your_user>/.open-interface_. Remember to replace "/" with "\" if copy pasting path into file explorer in Windows. Deleting settings.json will make the program reset the selected model and thus allow it to launch.

**Comment by AmberSahdev** [comment]
Fixed it for MacOS M Series in 0.6.1. Will upload fixed binaries for Linux/Windows later in the weekend.

**Comment by AmberSahdev** [comment]
New release better handles error during startup across all platforms 
https://github.com/AmberSahdev/Open-Interface/releases/tag/0.7.0
